tools/benchmark/src/ReporterUtilities.ts

```diff
@@ -40,6 +44,24 @@ export function getName(name: string): string {
 	return s.trim();
 }
 
+/**
+ * Tags and formats the provided Title from the supplied BenchmarkDescription to create a
+ * tagged and formatted Title for the Reporter.
+ * 
+ * @param args - see {@link BenchmarkDescription} and {@link Titled}
+ * @returns - a formatted tagged title from the supplied BenchmarkDescription
+ */
+export function qualifiedTitle(args: BenchmarkDescription & Titled): string {
```

Comment:
This file is not the right place for this function. This file is for common logic shared between reporters, and this is logic shared between runners and is not used by reporters.

Reply:
I thin "Configuration" might be a better place for this, since it's a utility for doing stuff with the configuration options.

Reply:
Hm.  I see - in that case, would it make sense to add a RunnerUtilities file? Or should I just export it from the mocha/runner file?

Reply:
adding `runnerUtilities` (we should use lower case for modules: the upper-case ones here are old) would be an ok place as well, but I think just using the existing Configuration.ts file seems cleanest for now. Either is fine with me.

Reply:
Weird, my comment got deleted, but Configuration.ts sounds good to me.

Reply:
Alright, done!